forem/forem
# Documentation for End to End Testing

## What type of PR is this? (check all applicable)


## Description

This documents end to end testing for the Forem web codebase.

## Related Tickets & Documents

#382, #12143

## QA Instructions, Screenshots, Recordings

1. Pull this branch down locally, e.g. via the GitHub CLI, `gh co 12459`
2. Run `yarn` to ensure dependencies are up to date.
3. Run `yarn gitdocs serve`
4. The documentation will be available at `http://localhost:8000`
5. Navigate to the **Testing/QA Guide** section. End to End Tests should be in that section.

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/833231/107035309-001b4d80-6786-11eb-9106-7e9dfad21964.png">

### UI accessibility concerns?

N/A

## Added tests?

- [x] No, and this is why: This PR is strictly documenation.

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Cyril from Archer getting a pile of papers on his desk](https://media.giphy.com/media/uZVoF5hwbjSuFe6w1A/giphy.gif)
